src/main/java/org/humancellatlas/ingest/project/web/ProjectController.java

```diff
@@ -203,4 +199,16 @@
         }
     }
 
+    @GetMapping(path = "projects/filter")
+    public ResponseEntity<PagedResources<Resource<Project>>> filterProjects(
+            @RequestParam(required = false) String search,
```

Comment:
Use a pojo for the search filter and the `@ModelAttribute` annotation. 
See:

- comment in the project service
- [Spring docs](https://docs.spring.io/spring-framework/docs/current/reference/html/web.html#mvc-ann-modelattrib-method-args)